src/js/module.js

```diff
@@ -67,12 +67,13 @@ iotjs_module_t.resolveFilepath = function(id, directories) {
     var dir = directories[i];
     var modulePath = dir + id;
 
-    // START: Temprorary fix for TizenRT
-    // See: https://github.com/Samsung/TizenRT/issues/320
     if (modulePath[0] !== '/') {
       modulePath = process.cwd() + '/' + modulePath;
     }
-    // END: Temprorary fix for TizenRT
+
+    if (process.platform === 'tizenrt' && modulePath.indexOf("..") != -1) {
```

Comment:
@haesik What about removing this condition and `normalizePath` in resolveModPath? `normalizePath` seems to be called for all platforms. For `tizenrt`, it is needed because it does not support `..`. For tizen (and perhaps linux), it should be normalized to avoid duplicated cache. We can call `normalizePath` in common entry point.

Reply:
If you want to remove the condition we need to test for all system and I think it's better to use linux system call for linux.. but now I just want to test only TizenRT.

Reply:
> If you want to remove the condition we need to test for all system
If you added `process.platform === tizenrt` because your PR is tested only in tizenrt, we can improve later PR.

> I think it's better to use linux system call for linux.
IMHO, I am not sure it is bottleneck. Furthermore, I am not sure how much linux system call would be faster.

Anyway, for this stage, LGTM